docs/docsite/rst/finding/content_types.rst

```diff
@@ -0,0 +1,65 @@
+.. _content_types:
+
+*********************
+Ansible Content Types
+*********************
+
+.. contents:: Topics
+
+
+This topic describes the Ansible content types indexed by the Ansible Galaxy server.
+
+
+.. _ansible_roles:
+
+Ansible Roles
+=============
+
+A role enables the sharing and reuse of Ansible tasks. It contains Ansible playbook tasks, plus all the
+supporting files, variables, templates, and handlers needed to run the tasks. A role is a complete unit of
+automation that can be reused and shared.
+
+In practical terms, a role is a directory structure containing all the files, variables, handlers, Jinja templates,
+and tasks needed to automate a workflow. When a role is created, the default directory stucture contains the following:
+
+.. code-block:: bash
```

Comment:
Consider using a nice visualization: https://twitter.com/sarah_edo/status/1039167305245048833
(in future, not this PR, of course)